### DISCUSSION TITLE
[2.3.10] Grafana Update Suggestion

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#2125] [Ideas] [answered]
Security Onion 2.3.10 currently uses Grafana version 7.0.5 which has a vulnerability.
So, could you update it to 7.1.0-beta 1 or later on next Security Onion release?

**Comment by dougburks** [comment, marked as answer]
I've created Issue https://github.com/Security-Onion-Solutions/securityonion/issues/2133.

Could you provide a link to more information about the vulnerability in Grafana 7.0.5?

**Reply by Masaya-A** [reply]
https://github.com/grafana/grafana/blob/master/CHANGELOG.md

7.1.0-beta 1 (2020-07-01)
Security fixes
Graph: Fix XSS vulnerability with series overrides #25401. Thanks to Rotem Reiss for reporting this.

**Reply by dougburks** [reply]
Thanks!
